doc/src/compute_sna_atom.rst

```diff
@@ -296,6 +299,14 @@ This option is typically used in conjunction with the *chem* keyword,
 and LAMMPS will generate a warning if both *chem* and *bnormflag*
 are not both set or not both unset.
 
+The keyword *bikflag* determines whether or not to expand the bispectrum
+rows of the global array returned by compute snap. If *bikflag* is set
+to *1* then the bispectrum row, which is typically the per-atom bispectrum
+descriptors :math:`B_{i,k}` summed over all atoms *i* to produce
+:math:`B_k`, becomes bispectrum rows equal to the number of atoms. Thus,
+the resulting bispectrum rows are :math:`B_{i,k}` instead of just
+:math:`B_k`.
```

Comment:
Add "In this case, the entries in the final column for these rows are set to zero."

Reply:
Done

Reply:
@athomps @charlessievers as a reminder. If you both consider this to be reading for merging, you need to assign the PR to me.